### PR TITLE
Update NEWS.md

### PR DESCRIPTION
**Pull request by cormullion**
A few changes to mollify the OCD of the one who opened issue #17478...

### CODE REVIEW

**Comment by JeffBezanson** [comment]
Thanks.

File: NEWS.md [hunk 1]
```diff
@@ -257,6 +257,21 @@ Library improvements
   * `Date` and `DateTime` values can now be rounded to a specified resolution (e.g., 1 month or
     15 minutes) with `floor`, `ceil`, and `round` ([#17037]).
 
+* File handling
+
+  * A new function `walkdir()` returns an iterator that walks the directory tree of a directory. ([#1765])
+
```
Comment (tkelman) [inline NEWS.md]: Note the end-of-line whitespace failure here and below. I also think this whole block (the non-empty lines of it anyway) should be indented to line up with the rest of the Library improvements section.

File: NEWS.md [hunk 2]
```diff
@@ -257,6 +257,22 @@ Library improvements
   * `Date` and `DateTime` values can now be rounded to a specified resolution (e.g., 1 month or
     15 minutes) with `floor`, `ceil`, and `round` ([#17037]).
 
+  * File handling
+
+    * A new function `walkdir()` returns an iterator that walks the directory tree of a directory. ([#1765])
+
```
Comment (tkelman) [inline NEWS.md]: still failing whitespace check here

Comment (cormullion) [inline NEWS.md]: Any clues? I know nothing about this. 😕

Comment (tkelman) [inline NEWS.md]: just delete the trailing whitespace so the blank lines are empty

**Pull request merged**